anaSbitReadout.py

```diff
@@ -0,0 +1,376 @@
+#!/bin/env python
+"""
+    Script to analyze sbitreadout data
+    By: Caterina Aruta (caterina.aruta@cern.ch) and Francesco Ivone (francesco.ivone@cern.ch)
+    """
+if __name__ == '__main__':
+    import ROOT as r
+    import numpy as np
+    import root_numpy as rp
+    import glob
+    import sys
+    import os
+    import time
+    from optparse import OptionParser
```

Comment:
Switch to using `ArgumentParser` from `argparse`.  Then define relevant arguments by hand instead of the blanket import in `anaoptions` (most of which are not suitable here).

Reply:
Done